proxy.go

```diff
@@ -177,6 +179,44 @@ func printAgentLogs(sock string) error {
 	return nil
 }
 
+func setupSigtermNotifier() chan os.Signal {
+	sigCh := make(chan os.Signal)
+	signal.Notify(sigCh, syscall.SIGTERM)
```

Comment:
How about make this `signalNotifier()` and defining a global:

```
const termSignal = syscall.SIGTERM
```

You can then refer to `termSignal` to avoid hard-coding `SIGTERM` three times (including in the error string).

Reply:
Sure !